assets/javascripts/views/search/search.coffee

```diff
@@ -68,8 +68,10 @@ class app.views.Search extends app.View
     @value = @input.value
 
     if @value.length
+      app.document.showSidebar(false)
```

Comment:
This would be an issue because the `showDocList` is triggered also on page load. We would then be in the situation where we would want 2 different results calling `toggleSidebar` from the same state.
1. Sidebar hidden - page loads -> call to `showDocList`-> call to `toggleSidebar` (here we would want to keep it hidden)
2. Sidebar hidden - user types into the searchbox -> call to `toggleSidebar` (here we would want to show the sidebar)

The only way we can difference this two actions using the `onInput` function. Or maybe am I missing something